packages/design-system/src/components/N8nText/Text.stories.js

```diff
@@ -7,7 +7,7 @@ export default {
 		size: {
 			control: {
 				type: 'select',
-				options: ['small', 'medium', 'large'],
+				options: ['mini', 'small', 'medium', 'large'],
```

Comment:
love it. thanks for adding. can you call this `xsmall` instead? would allow us to add `xxsmall` instead if we must

Reply:
there was unused option mini, but yeah, xsmall would make more sense

Reply:
ya coming from elementio had to use mini in a few places. hopefully can get rid of it eventually